### PR TITLE
PostgresAdapter

### PR DESCRIPTION
**Pull request by gbrener**
Add recipe for [PostgresAdapter](github.com/ContinuumIO/PostgresAdapter).

### CODE REVIEW

File: recipes/postgresadapter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - postgresql
+    - psycopg2
+    - toolchain
+
+  run:
+    - python 
+    - numpy
+    - pandas
+    - six
+    - ordereddict  # [py26]
```
Comment (bgruening) [inline recipes/postgresadapter/meta.yaml]: We do not build for py26 anymore afaik.

File: recipes/postgresadapter/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://github.com/ContinuumIO/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py>=36]
+  script: python setup.py install
```
Comment (bgruening) [inline recipes/postgresadapter/meta.yaml]: pip install is preferred nowadays

File: recipes/postgresadapter/meta.yaml (hunk 1 above)
Comment (gbrener) [reply recipes/postgresadapter/meta.yaml]: https://github.com/conda-forge/staged-recipes/pull/5141/commits/2e329a78315b903997be115d546f39c43ccbcd58

File: recipes/postgresadapter/meta.yaml (hunk 2 above)
Comment (gbrener) [reply recipes/postgresadapter/meta.yaml]: https://github.com/conda-forge/staged-recipes/pull/5141/commits/2e329a78315b903997be115d546f39c43ccbcd58

**Comment by gbrener** [comment]
Thank you for the feedback @bgruening . I made the changes you requested.

File: recipes/postgresadapter/meta.yaml (hunk 2 above)
Comment (gbrener) [reply recipes/postgresadapter/meta.yaml]: After making this change, the build failed with a cryptic error message, so I reverted back. Hopefully this isn't a problem.

**Comment by synapticarbors** [comment]
It looks like this might be ready to merge. @bgruening -- do you want to take a look at it a final time and make sure everything has been addressed from your original review?

File: recipes/postgresadapter/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - numpy 1.11*
+    - cython
+    - postgresql
+    - psycopg2
+    - toolchain
```
Comment (mariusvniekerk) [inline recipes/postgresadapter/meta.yaml]: Can we switch this to the new compilers?

**Pull request closed without merging**